### PR TITLE
[MODSOURMAN-705] Logging improvement

### PR DESCRIPTION
**Pull request by RomanChernetskyi**
## Purpose
Logging improvement

## Approach
Added new and changed old logs

### CODE REVIEW

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by Aliaksandr-Fedasiuk — COMMENTED** [review]
@RomanChernetskyi, could you add logs for this recommendation:
Controller endpoints or methods containing business must have at least two messages:
method enter: containing method name and incoming parameters, DEBUG level

File: mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java [hunk 1]
```diff
@@ -127,6 +131,7 @@ public void getChangeManagerJobExecutionsById(String id, String lang, Map<String
   @Override
   public void deleteChangeManagerJobExecutionsById(String id, String lang, Map<String, String> okapiHeaders,
                                                    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    LOGGER.debug("deleteChangeManagerJobExecutionsById:: Method is not implemented");
```
Comment (Aliaksandr-Fedasiuk) [inline mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java:134]: is the level "warn" preferable here?

Comment (RomanChernetskyi) [reply mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java]: yes, thanks, changed

File: mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java [hunk 2]
```diff
@@ -137,18 +138,18 @@ public Future<JobExecutionSourceChunk> update(JobExecutionSourceChunk jobExecuti
       pgClientFactory.createInstance(tenantId)
         .update(TABLE_NAME, jobExecutionChunk, new Criterion(idCrit), true, updateResult -> {
           if (updateResult.failed()) {
-            LOGGER.error("Could not update jobExecutionSourceChunk with id {}", jobExecutionChunk.getId(), updateResult.cause());
+            LOGGER.warn("update:: Could not update jobExecutionSourceChunk with id {}", jobExecutionChunk.getId(), updateResult.cause());
             promise.fail(updateResult.cause());
           } else if (updateResult.result().rowCount() != 1) {
             String errorMessage = String.format("JobExecutionSourceChunk with id '%s' was not found", jobExecutionChunk.getId());
```
Comment (Aliaksandr-Fedasiuk) [inline mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java:144]: could you add the method name to the error message?

**Review by Aliaksandr-Fedasiuk — APPROVED** [review]

File: mod-source-record-manager-server/src/main/java/org/folio/services/EventDrivenChunkProcessingServiceImpl.java [hunk 3]
```diff
@@ -38,7 +38,7 @@ public EventDrivenChunkProcessingServiceImpl(@Autowired JobExecutionSourceChunkD
 
   @Override
   protected Future<Boolean> processRawRecordsChunk(RawRecordsDto incomingChunk, JobExecutionSourceChunk sourceChunk, String jobExecutionId, OkapiConnectionParams params) {
-    LOGGER.debug("Starting to process raw records chunk with id: {} for jobExecutionId: {}. Chunk size: {}.", sourceChunk.getId(), jobExecutionId, sourceChunk.getChunkSize());
+    LOGGER.trace("processRawRecordsChunk:: Starting to process raw records chunk with id: {} for jobExecutionId: {}. Chunk size: {}.", sourceChunk.getId(), jobExecutionId, sourceChunk.getChunkSize());
```
[the remark this answers is not in the document]
Comment (RomanChernetskyi) [reply mod-source-record-manager-server/src/main/java/org/folio/services/EventDrivenChunkProcessingServiceImpl.java]: changed

File: mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java (hunk 2 above)
Comment (RomanChernetskyi) [reply mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java]: added

**Pull request merged**